unit-tests/test_numerical_integrator.py

```diff
@@ -0,0 +1,34 @@
+"""
+Tests the numerical integrator.
+"""
+from gusto import NumericalIntegral
+from numpy import sin, pi
+import pytest
+
+
+def quadratic(x):
+    return x**2
+
+
+def sine(x):
+    return sin(x)
+
+
+@pytest.mark.parametrize("integrand_name", ["quadratic", "sine"])
+def test_numerical_integrator(integrand_name):
+    if integrand_name == "quadratic":
+        integrand = quadratic
+        upperbound = 3
+        answer = 9
+    elif integrand_name == "sine":
+        integrand = sine
+        upperbound = 2*pi
```

Comment:
Could you integrate this to `pi` instead of `2*pi` so that the answer isn't 0? Just in case the whole thing is doing nothing we would get 0 as the answer so this makes the test extra-safe